src/dropbox-base.js

```diff
@@ -219,6 +267,9 @@ export class DropboxBase {
     if (state) {
       authUrl += `&state=${state}`;
     }
+    if (tokenAccessType !== 'legacy') {
```

Comment:
Is relying on the defaulting behavior intentional? In the future are we relying on switching the default token access type to `online`, for instance?

Reply:
In the case of legacy, we do not want to send this parameter so we are checking whether or not we add this header.  In the case of changing the default, we will still want to add this header so once legacy is removed, we will remove this check